src/metrics/review-coverage.ts

```diff
@@ -0,0 +1,293 @@
+/*
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import gql from 'graphql-tag';
+
+import {getOrgRepos} from '../common-queries';
+import {GitHub} from '../gql';
+import {PullRequestCommitsQuery, PullRequestCommitsQueryVariables} from '../gql-types';
+import {RepoCommitsQuery, RepoCommitsQueryVariables} from '../gql-types';
+import {RepoPRsCommitsQuery, RepoPRsCommitsQueryVariables} from '../gql-types';
+
+import {getReviewsForPullRequest} from './review-latency';
+
+export class ReviewCoverageResult {
+  commits: Map<string, ReviewedCommit>;
+
+  constructor(commits: Map<string, ReviewedCommit>) {
+    this.commits = commits;
+  }
+
+  summary() {
+    const count = this.numReviewed();
+    return `There are ${this.commits.size} commits of which ${
+        count} are reviewed. \nReview coverage is ${
+        Math.round(count / this.commits.size * 100)}%.`;
+  }
+
+  numReviewed(): number {
+    let numReviewed = 0;
+    this.commits.forEach((x) => x.reviewed ? numReviewed++ : '');
```

Comment:
Maybe prefer a regular for loop here? I think the inert empty string negative case of the ternary is a little funny to read. Could also do `numReviewed += x.reviewed ? 1 : 0` to use the ternary more straightforwardly? Also you could do `forEach(({reviewed}) => ... }`

Reply:
Changed to the ternary with `? 1 : 0`. Destructuring is fancy, changed, though I'm not sure about its readability since now there's just lots of parens.

Reply:
Yeah, tbh I kinda prefer a regular for loop and if here

Reply:
Done